xml/deploy_cephadm.xml

```diff
@@ -701,131 +701,293 @@ o- time_server ................................................ [enabled]
+         </para>
+<screen>
+podman run --name myregistry -p <option>REG_HOST_PORT</option>:5000 \
+ -v /var/lib/registry:/var/lib/registry \
+ -v /var/lib/registry/auth:/auth:z \
+ -e "REGISTRY_AUTH=htpasswd" \
+ -e "REGISTRY_AUTH_HTPASSWD_REALM=Registry Realm" \
+ -e REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd \
+ -v /var/lib/registry/certs:/certs:z \
+ -e "REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt" \
+ -e "REGISTRY_HTTP_TLS_KEY=/certs/domain.key" \
+ -e REGISTRY_COMPATIBILITY_SCHEMA1_ENABLED=true -d registry:2
+</screen>
+        </step>
+        <step>
+         <para>
+          Test secure access to the registry:
+         </para>
+<screen>
+&prompt.user;curl https://<option>REG_HOST_FQDN</option>:<option>REG_HOST_PORT</option>/v2/_catalog \
+ -u <option>REG_USERNAME</option>:<option>REG_PASSWORD</option>
+</screen>
+        </step>
+       </substeps>
+      </step>
+     </procedure>
+     <procedure>
+      <title>Configure the local registry and access credentials</title>
+      <step>
+       <para>
+        Configure the URL of the local registry:
+       </para>
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/registry set <replaceable>REG_HOST_URL</replaceable></screen>
+      </step>
+      <step>
+       <para>
+        Configure the user name and password to access the local registry:
+       </para>
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/username set <replaceable>REG_USERNAME</replaceable></screen>
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/password set <replaceable>REG_PASSWORD</replaceable></screen>
+      </step>
+      <step>
+       <para>
+        Run <command>ceph-salt apply</command> to update the &salt; pillar on
+        all minions.
+       </para>
+      </step>
+     </procedure>
+     <tip>
+      <title>Registry cache</title>
       <para>
-       Run <command>ceph-salt apply</command> to update the &salt; pillar on
-       all minions.
+       To avoid re-syncing the local registry when new updated containers
+       appear, you can configure a <emphasis>registry cache</emphasis>.
       </para>
-     </step>
-    </procedure>
-    <tip>
-     <title>Registry cache</title>
+     </tip>
      <para>
-      To avoid re-syncing the local registry when new updated containers
-      appear, you can configure a <emphasis>registry cache</emphasis>.
-<!-- 2021-02-09 tbazant: still looking for better solution
-      Find
-      more information about containers and air-gapped scenarios in
-      <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/#_deployment_in_an_air_gapped_environment"/>.
-      -->
+      Cloud Native Application Development and Delivery methods require a
+      registry and a CI/CD (Continuous Integration/Delivery) instance for the
+      development and production of container images. You can use a private
+      registry in this instance.
      </para>
```

Comment:
I'm not sure why the above paragraph is here. It seems to be talking about developing and producing container images, and I don't see how that's relevant to ceph deployments, where the user is not creating their own containers, but rather using the ones we provide.